cli/config/config.go

```diff
@@ -83,10 +83,14 @@ func LoadFromReader(configData io.Reader) (*configfile.ConfigFile, error) {
 	return &configFile, err
 }
 
+// TODO remove this temporary hack, which is used to warn about the deprecated ~/.dockercfg file
+var printLegacyFileWarning bool
```

Comment:
mutex

Reply:
Ah, 🤦 I was thinking about that initially (and check if there's multiple paths to this code). Yeah, let me add one